templates/claim_payment.html

```diff
@@ -0,0 +1,29 @@
+{% extends 'base.html' %}
```

Comment:
we need to use the magic link stuff like premio stage uses

Reply:
changed. testing changes above.